packages/server/src/storage/postgres.ts

```diff
@@ -28,7 +28,7 @@ export class PostgresConfig extends Config {
     @ConfigParam()
     tlsCAFile?: string;
 
-    @ConfigParam()
+    @ConfigParam("boolean")
```

Comment:
@BrunoBernardino FYI: Non-string params need to be marked as such, i.e.: "number" for numbers, "boolean" for bools etc. Simple arrays are possible, too via the `string[]` etc.

Reply:
Oooh, nice to know!